tests/kafkatest/services/performance/streams_performance.py

```diff
@@ -0,0 +1,132 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os.path
+import signal
+
+from ducktape.services.service import Service
+from ducktape.utils.util import wait_until
+
+from kafkatest.directory_layout.kafka_path import KafkaPathResolverMixin
+
+#
+# Class used to start the simple Kafka Streams benchmark
+#
+class StreamsSimpleBenchmarkService(KafkaPathResolverMixin, Service):
+    """Base class for simple Kafka Streams benchmark"""
+
+    PERSISTENT_ROOT = "/mnt/streams"
+    # The log file contains normal log4j logs written using a file appender. stdout and stderr are handled separately
+    LOG_FILE = os.path.join(PERSISTENT_ROOT, "streams.log")
+    STDOUT_FILE = os.path.join(PERSISTENT_ROOT, "streams.stdout")
+    STDERR_FILE = os.path.join(PERSISTENT_ROOT, "streams.stderr")
+    LOG4J_CONFIG_FILE = os.path.join(PERSISTENT_ROOT, "tools-log4j.properties")
+    PID_FILE = os.path.join(PERSISTENT_ROOT, "streams.pid")
+
+    logs = {
+        "streams_log": {
+            "path": LOG_FILE,
+            "collect_default": True},
+        "streams_stdout": {
+            "path": STDOUT_FILE,
+            "collect_default": True},
+        "streams_stderr": {
+            "path": STDERR_FILE,
+            "collect_default": True},
+    }
+
+    def __init__(self, context, kafka):
+        super(StreamsSimpleBenchmarkService, self).__init__(context, 1)
+        self.kafka = kafka
+        self.args = {} 
```

Comment:
`self.args` doesn't serve much of a purpose - may as well delete this and construct the `args` dict from scratch in `start_cmd` method (you're effectively doing so already).